ascii-dev/maintenance-tracker
# Fix Tests Not Passing & Travis Build Error

#### What does this PR do?
Work on fixing the blockers that were making tests fail. Also work on making sure TravisCI build passes.

#### How should this be manually tested?
After cloning the repo and running `npm install`, run `npm test` to run the tests and see them pass.

#### What are the relevant pivotal tracker stories?
#157924455
